doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -5,269 +5,297 @@ Reasoning with inductive types
-      clause is an occurrence clause whose syntax and behavior is described
-      in :ref:`occurrences sets <occurrencessets>`.
+   :n:`{+, @induction_clause }`
+     Giving multiple :n:`@induction_clause`\s is equivalent to applying :n:`destruct`
+     serially on each :n:`@induction_clause`.
 
-   .. tacv:: destruct @term {? with @bindings } {? as @or_and_intropattern_loc } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
-             edestruct @term {? with @bindings } {? as @or_and_intropattern_loc } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
+   :n:`@natural`
+     Equivalent to :tacn:`intros` :n:`until @natural` followed by `destruct` applied to
+     the last introduced hypothesis.
 
-      These are the general forms of :tacn:`destruct` and :tacn:`edestruct`.
-      They combine the effects of the ``with``, ``as``, ``eqn:``, ``using``,
-      and ``in`` clauses.
+     .. note::
+        To destruct a number, use the syntax :n:`destruct (@natural)` (not
+        very interesting anyway).
 
-.. tacn:: case @term
-   :name: case
+   :n:`@ident` (in :n:`@destruction_arg`)
+     If :token:`ident` is a quantified variable in the conclusion
+     of the goal, then :n:`destruct @ident` behaves like :tacn:`intros`
+     :n:`until @ident; destruct @ident`.
+     If :token:`ident` is a hypothesis of the context and :token:`ident`
+     is no longer dependent in the goal after application
+     of :tacn:`destruct`, it is erased (to avoid erasure, use parentheses, as
+     in :n:`destruct (@ident)`).
+
+   :n:`using @one_term {? with @bindings }`
+     Makes the tactic equivalent to :tacn:`induction` :n:`@induction_clause_list`.
+
+   :n:`@as_or_and_ipat`
+      Provides names for the variables introduced
+      into the context.  The :token:`as_or_and_ipat` must have the
+      form :n:`[ {*| {* @intropattern } } ]` with one :n:`{* @intropattern }`
+      for each constructor in the order in which the constructors are defined.
+      The items in
+      each :n:`{* @intropattern }` are used to name the variables created in
+      each new goal (one per constructor).  If there are not enough names,
+      Coq picks fresh names.
+
+   :n:`eqn : @naming_intropattern`
+      Defines new items in the context for the matching terms using the specified
+      or generated names.
+
+   :n:`with @bindings`  (in :n:`@one_term_with_bindings`)
+      Provides explicit instances for
+      the :term:`dependent premises <dependent premise>` of the type of
+      :token:`one_term`.
+
+   :n:`@occurrences`
+     Selects specific subterms of the goal and/or hypotheses to apply
+     the tactic to.  See :ref:`Occurrence clauses <occurrenceclauses>`.
+     If it occurs at the end of the :n:`@induction_clause_list`, then
+     there can only be one :n:`induction_clause`, which can't have its
+     own :n:`occurrences` clause.
```

Comment:
This bit goes away by applying my suggestion above of moving this trailing occurrence clause inside the `using` clause.

Reply:
I don't think so.  IIUC each `induction_clause` can have it's own `occurrences`, for example
`destruct n at 1, n at 1.` is valid.

I created #14510